frontend/apps/mobile/src/app/pages/page-manage-team/team-members/team-members.component.html

```diff
@@ -0,0 +1,25 @@
+
+
+<app-data-container [formGroup]="form">
+  <app-data-container-row>
+    <h5 ngProjectAs="container-headline">{{ 'page-manage-team.members' | translate }}</h5>
+  </app-data-container-row>
+
+  @if (form.controls.members.length > 0) { @for (member of
```

Comment:
Btw. für die nächsten Anpassungen:
Mann kann auch `@empty` anstatt des if else nutzen.

[Mehr Infos](https://angular.dev/guide/templates/control-flow#providing-a-fallback-for-for-blocks-with-the-empty-block)

Reply:
Wir haben btw. auch das track immer falsch gemacht...
Hier müssten wir eigentlich etwas, wie `track member.id` machen.
In den Docs steht auch, dass das die Performance stark erhöhen würde